templates/CRM/Event/Import/Form/MapField.tpl

```diff
@@ -51,7 +51,7 @@
    </tr>
    <tr>
      <td>
-  <div id="crm-submit-buttons">
+  <div class="crm-submit-buttons">
```

Comment:
Why some buttons have class and id and some only class?

Reply:
@pradpnayak that's structuring issue. We are trying to identify and fix any such bugs.

Reply:
@pradpnayak no reply from your end, should we move ahead and merge it?